GitUI/UserControls/RevisionGrid/RevisionGridControl.Designer.cs

```diff
@@ -437,6 +440,14 @@ private void InitializeComponent()
             this.openBuildReportToolStripMenuItem.Text = "Open build report in the browser";
             this.openBuildReportToolStripMenuItem.Click += new System.EventHandler(this.openBuildReportToolStripMenuItem_Click);
             // 
+            // openBuildReportToolStripMenuItem
+            // 
+            this.openPullRequestPageStripMenuItem.Image = global::GitUI.Properties.Images.PullRequest;
+            this.openPullRequestPageStripMenuItem.Name = "openPullRequestPageStripMenuItem";
+            this.openPullRequestPageStripMenuItem.Size = new System.Drawing.Size(301, 26);
+            this.openPullRequestPageStripMenuItem.Text = "Open pull request page in the browser";
```

Comment:
```suggestion
            this.openPullRequestPageStripMenuItem.Text = "Open pull request page in a browser";
```
may be even
```suggestion
            this.openPullRequestPageStripMenuItem.Text = "View pull request in a browser";
```

Reply:
accepted. But now it's not in line with the "Open build report in the browser". So I have updated it also...